.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+on:
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+
+      - name: Release with Maven
+        run: mvn clean -Dusername=$GIT_USER_NAME -Dpassword=GITHUB_TOKEN release:prepare release:perform
+        env:
+          GIT_USER_NAME: acrolinx-opensource
+          GIT_USER_EMAIL: opensource@acrolinx.com
+          GITHUB_ACTOR: ${{ github.actor }}
```

Comment:
who uses this `GITHUB_ACTOR`?

Reply:
I thought that the release:perform goal also deploys stuff, and the GITHUB_ACTOR is defined in the deploy workflow too.

Reply:
You are right, sorry my mistake, GITHUB_ACTOR and GITHUB_TOKEN need to be declared for the GitHub package registry.